distributed/distributed-schema.yaml

```diff
@@ -349,7 +349,12 @@ properties:
                   - string
                   - integer
                 description: |
-                  The maximum size of a message sent between workers
+                  The maximum amount of data for a worker to request from another in a single ``GatherDep``
+
+                  Tasks are gathered in batches, and if the first task in a batch is larger than this value,
+                  the task will still be gathered to ensure progress. Hence, this limit is not absolute. 
+                  Note that this limit applies to a single ``GatherDep`` and a worker may gather data from 
+                  multiple workers in parallel.
```

Comment:
The `GatherDep` class is an implementation detail and shouldn't appear in what is effectively functional documentation. Could you rephrase? e.g. replace \`\`GatherDep\`\`` with "data transfer between workers"?

Reply:
I've replaced `GatherDep` with _gathering operation_. This maps to the nomenclature we use in our documentation for the worker state machine. A data transfer _may_ be falsely interpreted as a transfer on the comm level, i.e., the transfer of an individual frame, which in turn has its own configuration variable.